src/DotVVM.Framework/Routing/DotvvmRouteTable.cs

```diff
@@ -87,7 +85,7 @@ public void AddGroup(string groupName, string urlPrefix, string virtualPathPrefi
         /// Creates the default presenter factory.
         /// </summary>
         public IDotvvmPresenter GetDefaultPresenter(IServiceProvider provider) =>
-            provider.GetRequiredService<IDotvvmPresenter>();
+            group?.PresenterFactory(provider) ?? provider.GetRequiredService<IDotvvmPresenter>();
```

Comment:
In this case you accept `null` returned from `PresenterFactory`, when the presenter is specified for a single route, null result is not accepted. I think this behavior should be unified.